app/views/admin/invitations/index.html.erb

```diff
@@ -66,20 +66,24 @@
       </tr>
     </thead>
     <tbody>
-      <% @invitations.each do |user| %>
-        <tr>
-          <td class="pl-7">
-            <div class="flex items-center">
-              <%= render "admin/users/index/member_image", user: user %>
-              <span class="color-base-60">@<%= user.username %></span>
-            </div>
-          </td>
-          <td><%= user.email %></td>
-          <td><%= user.created_at.strftime("%d %b, %Y") %></td>
-          <td>
-            <%= render "admin/users/index/invitation_actions_dropdown", user: user, context: "large" %>
-          </td>
-        </tr>
+      <% if @invitations.any? %>
+        <% @invitations.each do |user| %>
+          <tr>
+            <td class="pl-7">
+              <div class="flex items-center">
+                <%= render "admin/users/index/member_image", user: user %>
+                <span class="color-base-60">@<%= user.username %></span>
+              </div>
+            </td>
+            <td><%= user.email %></td>
+            <td><%= user.created_at.strftime("%d %b, %Y") %></td>
+            <td>
+              <%= render "admin/users/index/invitation_actions_dropdown", user: user, context: "large" %>
+            </td>
+          </tr>
+        <% end %>
+        <% else %>
+          <%= render "admin/invitations/empty_state" %>
```

Comment:
So, I'm running into something interesting here... I am unable to place the "empty state" where I need it (centered within the table body, beneath the table headings, per the design). I am continually running into a situation where the "empty state" is placed above the table headings, centered and looking as expected, _or_ it is within the table body, but is not centered and does is not looking as expected--I've included screenshots below for reference:

**Placed above the table headings, centered and looking as expected:**
![Screen Shot 2022-05-04 at 1 44 22 PM](https://user-images.githubusercontent.com/32834804/166814682-979c0b08-a836-457c-8796-546a0ec2eebf.png)

**Within the table body, but not centered or looking as expected:**
![Screen Shot 2022-05-04 at 1 44 02 PM](https://user-images.githubusercontent.com/32834804/166814702-3b877f11-4d81-486a-956b-740da96ae046.png)

Any thoughts on how to get around this? I've tried different stylings, moving the "empty state" around in the view, removing and adding table rows, etc. to no avail. It's clearly the table that is causing issues, as the "empty state" looks as expected when it renders outside of the table columns/rows, but the design shows the "empty state" within the table, beneath the table headers. 🤔

Reply:
Quick question, is the empty state partial re-used?  If it is where is it used?  If not let's consider inlining it for this exercise.

My assumption is that we'd have something like:

```html
<tr>
  <td colspan="4">
    <div class="align-center flex flex-col my-auto py-7">
       <p class="fs-xl fw-normal mb-2">No members invited yet.</p>
       <p class="color-secondary">Invite members and grow your community.</p>
    </div>
    <div class="flex grow-1 justify-content-center">
       <%= link_to "Invite member", new_admin_invitation_path, class: "c-cta c-cta--branded ml-3" %>
    </div>
  </td>
</tr>
```

Looking at MDN docs for tables, the allowed elements are `caption`, `colgroup`, `thead`, `tbody`, `tr` and `tfoot` (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table)

So, you'll need a `tr` tag.  Which ([per MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr)) needs a `td` tag.  The `td` tag can then have flow elements (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td); the flow content are `div` and `p`-tags (and more).

Reply:
Just catching up with this now, and I think what we want to do is have the empty state message _outside_ of the table completely. Putting the empty state inside the table itself causes conflicting info for users of assistive tech - for the example above we'd announce we had 1 row of data in the table, and the invited member username is "No members invited yet..." etc.

TL;DR, let's move this empty state chunk to below the table itself. As a bonus we could even link the empty state message with the table by giving it an ID and then using `aria-describedby="empty-state-message-id` on the `<table>` element 🙂

Reply:
Sounds good; will do!